print/resources/inkstitch.js

```diff
@@ -735,6 +735,7 @@ $(function() {
     settings["operator-overview"] = $("[data-field-name='operator-overview']").is(':checked');
     settings["operator-detailedview"] = $("[data-field-name='operator-detailedview']").is(':checked');
     settings["operator-detailedview-thumbnail-size"] = $("[data-field-name='operator-detailedview-thumbnail-size']").val();
+    settings["pattern-onlyview"] = $("[data-field-name='pattern-onlyview']").val();
```

Comment:
Change this to `settings["pattern-onlyview"] = $("[data-field-name='pattern-onlyview']").is(':checked');`